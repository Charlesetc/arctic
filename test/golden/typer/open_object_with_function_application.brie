[
  define x x
  x.this a
]
