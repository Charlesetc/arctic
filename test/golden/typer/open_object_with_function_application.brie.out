{[3]=>Open_function, [0, 1, 2, 4]=>Open_object(fields={"this"=>3})}